frontend/src/components/AccountDetailsEditContainer.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div>
+    <ContainerComponent>
+      <template #Header>
+        <h1 class="underline text-xl">Daten ändern:</h1>
+      </template>
+
+      <template #default>
+        <div class="text-gray-400 p-1 text-xs">
+          Hinweis: Sie müssen nicht alle Felder ausfüllen, leere Felder werden
```

Comment:
I dont think this is a good thing. 
Many users wont skip these texts escpecially since they are grayed out and by that seem "irrelevant".
So when the field is empty a uer would expect to be able to set the field to empty

Reply:
Since fields are pre-filled already anyways, I dont think this notice is needed anymore